boilerplate/scripts/build.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+#
+# Build with current local code:
+#   ./build.sh
+#
+# Specify core and legacy-scripting-runner versions to build with published
+# packages on npm:
+#   ./build.sh 9.0.0 3.2.4
+
+update_deps() {
+    package_json=$1
+    core_version=$2
+    legacy_version=$3
+
+    cmd_for_core="s|\"zapier-platform-core\": \"[^\"]*\""
+    cmd_for_core+="|\"zapier-platform-core\": \"$core_version\"|g"
+
+    cmd_for_legacy="s|\"zapier-platform-legacy-scripting-runner\": \"[^\"]*\""
+    cmd_for_legacy+="|\"zapier-platform-legacy-scripting-runner\": \"$legacy_version\"|g"
+
+    sed -i.bak "$cmd_for_core" "$package_json"
+    sed -i.bak "$cmd_for_legacy" "$package_json"
```

Comment:
Originally, we have a Node script [packages/core/bin/update-boilerplate-dependencies.js](https://github.com/zapier/zapier-platform/blob/2659f50ed953ce231012ed57d410936db3b84204/packages/core/bin/update-boilerplate-dependencies.js) that does the dep version replacement in package.json. I deleted the Node.js in favor of this `sed` one-liner.